grim.go

```diff
@@ -146,6 +151,11 @@ func (i *Instance) BuildNextInGrimQueue(logger *log.Logger) error {
 	return nil
 }
 
+func repoIsAlive(hook *hookEvent) bool {
+	url := hook.URL
+	return len(url) >= 12 && url[len(url)-12:] != "000000000000"
```

Comment:
Its certainly clever but I think there may be a better way.  The docs suggest there is a boolean called "deleted" in the JSON.  The docs don't bother to explain it though.  It will take some testing to verify it is the thing I think it is.

https://developer.github.com/v3/activity/events/types/#pushevent